server/datastore/mysql/fulltext.go

```diff
@@ -5,7 +5,7 @@ import (
 	"strings"
 )
 
-var mysqlFTSSymbolRegexp = regexp.MustCompile("[-+]+")
+var mysqlFTSSymbolRegexp = regexp.MustCompile("[-+@]+")
```

Comment:
Double checking to see if I understand the issue and the fix:

The issue is that the `@` character causes a MySQL error when using it on a `MATCH` statement. This change will get rid of it before using it on the `MATCH` statement (for searching labels only).

Right?

If so, I think the fix is a-ok, as the likelihood for a user to wish to match "@" on a label name is pretty low.

Reply:
Yup, pretty much. 

The only thing to keep in mind is that with this the search will return less specific results, for example, If we have two labels: `foo@bar` and `bar@foo`, and I search for `foo@bar` it will return both results.